testing/file_test/file_test_base_test.cpp

```diff
@@ -145,49 +149,64 @@ static auto TestFailExample(TestParams& params)
     -> ErrorOr<FileTestBaseTest::RunResult> {
   int msg_count = 0;
-  params.stdout << "unattached message " << ++msg_count << "\n"
-                << "file: a.carbon\n"
-                << "unattached message " << ++msg_count << "\n"
-                << "line: 3: attached message " << ++msg_count << "\n"
-                << "unattached message " << ++msg_count << "\n"
-                << "line: 8: late message " << ++msg_count << "\n"
-                << "unattached message " << ++msg_count << "\n"
-                << "file: b.carbon\n"
-                << "line: 2: attached message " << ++msg_count << "\n"
-                << "unattached message " << ++msg_count << "\n"
-                << "line: 7: late message " << ++msg_count << "\n"
-                << "unattached message " << ++msg_count << "\n";
+  params.output_stream << "unattached message " << ++msg_count << "\n"
+                       << "file: a.carbon\n"
+                       << "unattached message " << ++msg_count << "\n"
+                       << "line: 3: attached message " << ++msg_count << "\n"
+                       << "unattached message " << ++msg_count << "\n"
+                       << "line: 8: late message " << ++msg_count << "\n"
+                       << "unattached message " << ++msg_count << "\n"
+                       << "file: b.carbon\n"
+                       << "line: 2: attached message " << ++msg_count << "\n"
+                       << "unattached message " << ++msg_count << "\n"
+                       << "line: 7: late message " << ++msg_count << "\n"
+                       << "unattached message " << ++msg_count << "\n";
   return {{.success = true}};
 }
 
 // Does printing and returns expected results for file_only_re_one_file.carbon.
 static auto TestFileOnlyREOneFile(TestParams& params)
     -> ErrorOr<FileTestBaseTest::RunResult> {
-  params.stdout << "unattached message 1\n"
-                << "file: file_only_re_one_file.carbon\n"
-                << "line: 1\n"
-                << "unattached message 2\n";
+  params.output_stream << "unattached message 1\n"
+                       << "file: file_only_re_one_file.carbon\n"
+                       << "line: 1\n"
+                       << "unattached message 2\n";
   return {{.success = true}};
 }
 
 // Does printing and returns expected results for no_line_number.carbon.
 static auto TestNoLineNumber(TestParams& params)
     -> ErrorOr<FileTestBaseTest::RunResult> {
-  params.stdout << "a.carbon: msg1\n"
-                   "msg2\n"
-                   "b.carbon: msg3\n"
-                   "msg4\n"
-                   "a.carbon: msg5\n";
+  params.output_stream << "a.carbon: msg1\n"
+                          "msg2\n"
+                          "b.carbon: msg3\n"
+                          "msg4\n"
+                          "a.carbon: msg5\n";
+  return {{.success = true}};
+}
+
+// Does printing and returns expected results for stdin.carbon.
```

Comment:
Had a minor hiccup reading this as starting a question ("Does printing <do this?>") rather than descriptive. Ambiguity might be removed by rephrasing to "Prints and returns ... "?

Reply:
Done